### PR TITLE
Fix for sidebar vertical line issue

### PR DESCRIPTION
**Pull request by marjohn56**

### CODE REVIEW

File: src/opnsense/www/themes/opnsense/assets/stylesheets/main.scss [hunk 1]
```diff
@@ -478,6 +478,7 @@ a.list-group-item:focus {
   text-decoration: none !important;
   color: #ec6d12 !important;
   background-color: lightergrey !important;
+  border: none! important;
```
Comment (AdSchellevis) [inline src/opnsense/www/themes/opnsense/assets/stylesheets/main.scss]: @marjohn56 shouldn't this be `none !important` ? de space looks misplaced.

Comment (opnsenseuser) [reply src/opnsense/www/themes/opnsense/assets/stylesheets/main.scss]: My fault

**Comment by marjohn56** [comment]
Doh... my fault René, I should have looked at what I was doing. Fixed now

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request closed without merging**